docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx
-sphinx_rtd_theme
\ No newline at end of file
+pydata-sphinx-theme==0.14.3
+sphinx==7.3.7
```

Comment:
Can we unpin here as well, for reasons discussed on other PRs?

Reply:
Unpinned